src/Mailer/Email.php

```diff
@@ -1830,6 +1841,29 @@ public function message($type = null)
     }
 
     /**
+     * Sets priority.
+     *
+     * @param int|null $priority 1 (highest) to 5 (lowest)
+     * @return $this
+     */
+    public function setPriority($priority)
+    {
+        $this->_priority = $priority;
```

Comment:
Seems like we should have some validation here. Right now a hapless user could set `'boogers'`. Seems like a mistake we could catch and warn them about.